python/test/conftest.py

```diff
@@ -0,0 +1,83 @@
+import numpy as np
+import pytest
+import state_representation as sr
+
+
+class Helpers:
+    @staticmethod
+    def assert_cartesian_equal(state1: sr.CartesianState, state2: sr.CartesianState, test_state_attributes=True):
+        if test_state_attributes:
+            assert state1.get_name() == state2.get_name()
+            assert state1.get_type() == state2.get_type()
+            assert state1.is_empty() == state2.is_empty()
+        assert state1.get_reference_frame() == state2.get_reference_frame()
+        if state1:
+            assert state1.dist(state2) < 1e-4
```

Comment:
nitpick: it would be nice to provide the tolerance as an argument instead of hard-coding it, here and in joint state. Just as good practice.

```python
    def assert_equal(..., tolerance = 1e-4):
        ...
        assert state1.dist(state2) < tolerance
```